docs/include/codec.asciidoc

```diff
@@ -0,0 +1,41 @@
+[id="plugins-{type}s-common-options"]
```

Comment:
I think we can remove this file. Codecs don't need any base docs like this. I think it is an accident that codecs have enable_metrics and id settings and we can remove that feature in the future, but for users, they don't need this.